### DISCUSSION TITLE
Placeholder when a mission's category is completed

### DISCUSSION THREAD

**Discussion by Qg9** [BG-Software-LLC/SuperiorSkyblock2#1104] [Implemented Feature Requests] [answered]
Something like ASkyblock in old time, when all the missions of a category is completed, the output is true

**Comment by OmerBenGera** [comment, marked as answer]
Added in latest dev build. Use `%superior_player_missions_completed_<category>%` to retrieve the amount of completed missions in a specific category. Instead of checking if it's true, you can check if it's the same as the amount of missions in the category :)
